02.general/09.license-key/docs.en.md

```diff
@@ -16,13 +16,16 @@ visible: true
 <a name="key"></a>
 #### What is AdGuard license key?
 
-License key is a unique symbol sequence that activates AdGuard. License key is required for AdGuard’s work after trial period's expiration. There are few different types of licenses available:
+License key is a unique symbol sequence that activates AdGuard. License key is required for AdGuard’s work after trial period's expiration. There are three types of licenses available:
 
-1. **Standard** - allows you to activate AdGuard on Windows PC or Mac.
-2. **Mobile** - allows you to activate AdGuard on your Mobile device.
-2. **Premium** - mobile and standard in a single license. Allows you to activate both mobile device and computer with the same license key.
-4. **Bonus License** - is given for winning any contests or prize draws. Equivalent to standard license.
-5. **Beta-test** - special type of license for AdGuard beta-testers. Program activated with this type of license is automatically updated to the latest available beta.   
+1. **Personal** - This license key allows you to activate AdGuard on any **three** devices.
+
+2. **Family** is also a universal type of license, the only difference is the number of devices. By purchasing this key you become the owner of **nine** desired licenses for your devices.
+
+3. **Beta-test** - special type of license for AdGuard beta-testers. Program activated with this type of license is automatically updated to the latest available beta.   
+
+**Please NOTE**: All types of licenses are universal. You can activate any of the presented devices:
```

Comment:
`>**Please notice**: all types of licenses are universal. This means you can use them to activate any AdGuard products in any combination.`